appveyor.yml

```diff
@@ -61,7 +64,14 @@ build_script:
     ) else (
       cmake --build . --target install --config Release
     )
-  - cd ..
+  - cd C:\boost_1_64_0
+  - set boost_dir=C:\boost_1_64_0
+  - set cores=%NUMBER_OF_PROCESSORS%
```

Comment:
I was going to say don't bother because CI only has one core anyway but I was wrong and there are actually two cores according to https://www.appveyor.com/docs/build-environment/#build-vm-configurations.

We should probably make use of this for all other builds as well.

Reply:
OK! I didn't know it. It worked on my Windows machine, so I added it here too :). Thanks for the info!